crates/engine/tree/src/tree/mod.rs

```diff
@@ -3417,7 +3420,8 @@ mod tests {
     async fn test_tree_state_remove_before_finalized() {
         let start_num_hash = BlockNumHash::default();
         let mut tree_state = TreeState::new(start_num_hash);
-        let blocks: Vec<_> = TestBlockBuilder::default().get_executed_blocks(1..6).collect();
+        let blocks: Vec<_> =
+            TestBlockBuilder::<EthPrimitives>::default().get_executed_blocks(1..6).collect();
```

Comment:
I'd like to introduce a `TestBlockBuilder::eth` function for this

Reply:
thank, i just updated